instances/terminology/differenceMeasure.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/terminology/differenceMeasure",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Terminology",
+  "definition": "A measure of the difference between two things",
+  "description": "This may be a numerical or physical quantity, a set of categories, etc. Examples include 'mean squared error', 't-statistic', 'p-value'.",
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "measured type",
```

Comment:
I overlooked this one... name is "difference measure"

Reply:
done